### ISSUE TITLE
508 Accessibility: CSS should not remove focus outline from buttons

### ISSUE THREAD

**Issue by cjfreedman** [asyncapi/asyncapi-react#961]
We use a tool called Sort Site to evaluate our site for Section 508 Accessibility compliance.  It is flagging the following issue, where you can see in the code that it has the property "focus:outline-none"

![image](https://github.com/asyncapi/asyncapi-react/assets/86489380/31a29ca5-ea04-410e-bb36-7ca410dbe6c1)

On our page, this is on the Payload button:
![image](https://github.com/asyncapi/asyncapi-react/assets/86489380/e4acaf94-7c7c-48eb-bbcf-b547ba38850c)

All buttons should have a focus outline, the focus outline should not be removed.

**Comment by Souvik9205** [comment]
Hey, I want to work on this issue!

**Comment by jangala-laxman** [comment]
I would like to work on this issue, can you assign this to me please.

**Comment by Aditya90456** [comment]
i working on this

**Comment by blazethunderstorm** [comment]
@AceTheCreator can ii work on this issue ?

**Comment by mitaliiiiii** [comment]
can this be assigned to me ?
